notebooks/_06-Architecture.md

```diff
@@ -304,9 +332,15 @@ The only "website" for the project will be its github repository.
 
 * All the plots will be generated using functions from the source code. Additional plots as needed will be generated
 using matplotlib/seaborn. 
+    - The plots generated will show the distribution of the recovered parameters and where the true parameters fall within that distribution. Or where the recovered parameters fall within the confidence intervals of the "true" parameters assumed in simulating the data.
```

Comment:
Your second sentence sounds hard since the data-generating parameters have no confidence interval around them. One only gets confidence intervals around estimated parameters.

For the second sentence, I would instead show the proportion of estimates where the true parameters fall within some percentile confidence interval (e.g. 95th percentile) around the estimate. This would essentially just end up being a bar plot with two bars or a table.

Reply:
I guess what I meant by "true" parameters is the parameters used to simulate the data. 
Making the change now.